contracts/IRelayRecipient.sol

```diff
@@ -30,6 +32,19 @@ contract IRelayRecipient {
      */
     function acceptRelayedCall(address relay, address from, bytes memory encodedFunction, uint gasPrice, uint transactionFee, bytes memory approval) public view returns (uint);
 
```

Comment:
@drortirosh  already fixed some comments in the PR #128, so I tried avoiding changing much where he already did . Once @drortirosh 's PR is merged I will rebase and fix some of the comments